ui.go

```diff
@@ -286,29 +286,21 @@ func (win *Win) printd(dir *Dir, marks map[string]bool) {
 			s = append(s, make([]rune, win.w-2-w)...)
 		}
 
-		switch gOpts.showinfo {
-		case "none":
-			break
-		case "size":
-			if win.w > 8 {
-				h := humanize(f.Size())
-				s = runeSliceWidthRange(s, 0, win.w-3-len(h))
-				s = append(s, ' ')
-				for _, r := range h {
-					s = append(s, r)
+		if gOpts.showinfo != nil {
+			var tmp string
```

Comment:
@IvanMenshykov A minor nitpick, I think you can name this `info` instead. I think `tmp` is a non-informative name that should be avoided as much as possible. I agree though the rest of the codebase is not the best example for proper naming :)